### ISSUE TITLE
is there a way to add books that are a collection of mp3's to be able to have all my books in one place?

### ISSUE THREAD

**Issue by mweske** [rmcrackan/Libation#718]
I could not figure out any other way to ask this question.
I have a collection of books on CD that I have ripped to be able to listen on devices without a CD drive.
Is there a way to add those books that are a collection of mp3's to libation so all my books with we in one location?
If there is something in the documentation, I missed it.
Thank you.

**Comment by CLHatch** [comment]
> I could not figure out any other way to ask this question. I have a collection of books on CD that I have ripped to be able to listen on devices without a CD drive. Is there a way to add those books that are a collection of mp3's to libation so all my books with we in one location? If there is something in the documentation, I missed it. Thank you.

Not really what this app is meant to do.  This app is for downloading any books you've purchased from Audible to your computer.  But if you have a machine you can use as a "server", I highly recommend https://github.com/advplyr/audiobookshelf

**Comment by rmcrackan** [comment]
tl;dr: this is not currently available and probably is not going to happen

Longer answer: Although Libation has a lot of itunes-like look and feel, its focus is entirely different. We take the books you paid for in audible and give them to you as files you can use anywhere. Many users even immediately move their books to another place that Libation doesn't even know about and we encourage this.

Here's where it breaks down. All of the little hidden details in this program assume that these are audible books. So I could create a new type of ID for imported books and have a way to import your other book data; this wouldn't even be that difficult. Next, something (author, narrator, etc.) isn't going to match so we need to build a metatag editor. Now that we're starting to recreate itunes or mediamonkey (btw -- I love mediamonkey) the next thing is to make sure we can find moved, renamed, or edited files. The last step of the evolution will be to have a player. It can't be a normal player though because we need to support audiobook ideas like skipping chapters, remembering your place, and speed control.

All of these are great ideas. Really great ideas. They're also years of development time. More importantly though: I think in every case, someone else already does it better than I could.

I believe that Libation is the absolute best at what it does. It's my goal to give you the freedom to use your book files with any other software which is the best at what they do.

If there's something audiobook specific that someone else doesn't do better, I think there's a strong case for considering it for Libation.

**Comment by mweske** [comment]
CLHatch  Thank you.   When I get my Linux system rebuilt I will install audiobookshelf.  In the mean time I will use the "docker" version to install on my Windows 11 system.  Not thrilled about docker as it suckes up a lot of resources, but for a short term it should work.  I just have to read up on the needed format for the install.

**Comment by CharlieRussel** [comment]
@mweske -- Have you tried using the Windows Subsystem for Linux to load audiobookshelf? I haven't tried it myself, but it might be an interesting solution? Of course, if you're already rebuilding your Linux system, then probably unnecessary. But these days I'm always trying to consolidate my machines as much as possible.

**Comment by rmcrackan** [comment]
This is one of a number of far-reaching features that have been given a lot of thought for a long time with no commitments. I think it's fair to say we're not intending to do it in the near term.
